<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>20</width>
     <height>581</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="double_spin_box_info">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>510</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_box">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>480</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_13">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>460</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="spin_box_change">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>540</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QPushButton" name="double_spin_box_change">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>540</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spin_box_info">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="double_spin_box">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>480</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>20</width>
     <height>581</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
